src/RA_Dlg_MemBookmark.cpp

```diff
@@ -957,12 +905,12 @@ BOOL Dlg_MemBookmark::EditLabel(int nItem, int nSubItem)
         return FALSE;
     };
 
-    SendMessage(g_hIPEEditBM, WM_SETFONT, (WPARAM)GetStockObject(DEFAULT_GUI_FONT), TRUE);
-    SetWindowText(g_hIPEEditBM, NativeStr(m_vBookmarks[nItem]->Description()).c_str());
+    SetWindowFont(g_hIPEEditBM, ::GetStockObject(DEFAULT_GUI_FONT), TRUE);
```

Comment:
This message cracker casts everything so it doesn't really matter if you use `GetStockObject `or `GetStockFont`. It does expect `HFONT`, but the value is "unboxed".